app/assets/javascripts/dashboard/views/dash/transfer_task_modal.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+
+import _ from "lodash";
+import React from "react";
+import { Spin, Modal, Button } from "antd";
+import Request from "libs/request";
+import type { APIUserType } from "admin/api_flow_types";
+
+class TransferTaskModal extends React.PureComponent {
+  props: {
+    onChange: Function,
+    // Somehow, eslint doesn't recognize that annotationId is used in
+    // the async functions
+    // eslint-disable-next-line react/no-unused-prop-types
+    annotationId: ?string,
+    onCancel: Function,
+    visible: boolean,
+    userID: ?string,
+  };
+
+  state: {
+    isLoading: boolean,
+    users: Array<APIUserType>,
+    currentUserValue: string,
```

Comment:
Yeah, it's an id. I picked `value` because it is used in a `select` element, but I renamed it to `currentUserIdValue` now.